add_events_discord.py

```diff
@@ -0,0 +1,59 @@
+
+import discord
+from datetime import datetime
+from TournamentPuller import TournamentPuller
```

Comment:
```suggestion
from tournament_puller import TournamentPuller
```

Update since I renamed the file

Reply:
Bet I'll change this when I get home